### DISCUSSION TITLE
회의실 뿐만 아니라 모든 공간에서 private meeting이 가능했으면 좋겠습니다.

### DISCUSSION THREAD

**Discussion by IronKoh** [metamong-friends/metamong-friends.github.io#36] [제안하기]
안녕하세요.

런던 메트로 회의실을 회의실 용도로만 쓰지 않고 있습니다.
회의 뿐만 아니라 스탠드업 장소로도 사용하고 있고, 개발팀이 상시 대기하는 공간으로도 사용 중입니다.

그 안에서 private 하게 공유가 필요한 경우가 많은데
그 때마다 위치를 회의실 밖으로 이동하여 사람을 불러내야 해서 불편합니다.

전체 장소에서 private meeting이 되었으면 좋겠습니다.

**Comment by ACT-TJ** [comment]
안녕하세요!

Project View 의 사용 시나리오를 잘 알고 있기에. 공감하고, 저희도 해당 상황에서 어떤 부분을
중점으로 둘 지 많은 고민을 하였습니다. 

전체 저희가 정의한 공간 분류와 각 공간별 기능정의가 되어있는 상태라 단순 기능의 제공 영역이
되긴 어렵고 저희 공간에 대한 정의를 검토해야 하는 상황입니다.

회의실 외 업무공간을 그룹화화 하여 사용 팀 안에서도 그룹으로 영역을 묶어 업무수행을 하실 때
활용하실 수 있는 공간 정의를 검토하고 있습니다.

개선 의견으로 분류하고 위 공간에 대한 개선을 진행할 때 주요 안건으로 논의를 거쳐 정리된 의견
드릴 수 있도록 하겠습니다.

좋은 의견, 공감 감사드립니다!
